invenio_communities/config.py

```diff
@@ -109,6 +115,11 @@
 ]
 """Community roles."""
 
+COMMUNITIES_SEARCH = {
+    'facets': ['type', 'domain'],
```

Comment:
COMMUNITIES_FACETS also contains visibility (see [here](https://github.com/inveniosoftware/invenio-communities/blob/master/invenio_communities/config.py#L44)), I guess it should be added here or removed from the facets.

Reply:
I'm a bit unsure about this, I tried to keep the same facets as the one that were previously hard-coded: https://github.com/inveniosoftware/invenio-communities/pull/547/files#r857564267

Reply:
There might be some things in the code that are not the best it could be and hardcoded things are usually a red flag for this, so if we can improve something we should take the opportunity :wink: 
As far as I understand and if I am not mistaken if it's declared in the COMMUNITIES_FACETS this means that it will generate the aggregation for the field (in this case `visibility`) and in the COMMUNITIES_SEARCH we establish what we want  to display in the search, if we do the aggregation I think we should use it, otherwise we should not do it.

Reply:
As discussed, to keep only `type`